model/formComponentInput.tsx

```diff
@@ -0,0 +1,9 @@
+import { EpisodeItem } from "./episodeModel";
+
+interface FormComponentInput {
```

Comment:
Make this a single interface with optional initData parameter

Reply:
fixed